public method foo() throws FooException {
	...
}